ValveResourceFormat/Resource/ResourceTypes/Texture.cs

```diff
@@ -449,7 +449,7 @@ private void SkipMipmaps()
                 }
                 else
                 {
-                    offset = CalculateBufferSizeForMipLevel(j);
+                    offset = CalculateBufferSizeForMipLevel(j) * (Flags.HasFlag(VTexFlags.CUBE_TEXTURE) ? 6 : 1);
```

Comment:
Interesting. Does this work for other encodings like dxt5?

Reply:
Yes. This envmap on my shots is DXT5.

Reply:
Okay, I tested it on Dota, and it does produce a better texture!